rt-scripts-optimizer.php

```diff
@@ -328,9 +348,26 @@ function disable_emojis_remove_dns_prefetch( $urls, $relation_type ) {
 /**
  * Enqueues loadCSS scripts.
  */
-function loadcss_scripts() {
+function rt_scripts_optimizer_load_scripts() {
 
 	wp_enqueue_script( 'loadCSS', RT_SCRIPTS_OPTIMIZER_DIR_URL . '/assets/js/loadCSS.min.js', array(), filemtime( RT_SCRIPTS_OPTIMIZER_DIR_PATH . '/assets/js/loadCSS.min.js' ), false );
 
 }
-add_action( 'wp_enqueue_scripts', 'loadCSS_scripts' );
+add_action( 'wp_enqueue_scripts', 'rt_scripts_optimizer_load_scripts' );
+
+/**
+ * Rename src attribute of iframes to block them from automatically loading on page load.
+ *
+ * This will be loaded by javascript.
+ *
+ * @param string $content Original content.
+ *
+ * @return string Modified content.
+ */
+function rt_scripts_optimizer_iframe_lazy_loading( $content ) {
+
+	return preg_replace( '~<iframe[^>]*\K (?=src=)~i', ' data-', $content );
```

Comment:
Please make it specific to youtube iframes not all iframes. This will affect other iframes as well such as Ads, thirdparty iframe tags, etc.